### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by edewit**
changed to use `cordova-plugin-test-framework` instead of `grunt-cordova-plugin-jasmine` making it more standard

to test execute the steps in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,19 @@
...
 
-To run tests you need [Grunt](http://gruntjs.com/getting-started#installing-the-cli) installed on your machine and then
-just open terminal in the root directory of this project and run `grunt test`, which will run the Jasmine tests located
-in `/tests` directory.
+1. Use your existing cordova app, or create a new one.
+1. Add the plugin and the tests:
+
+  ```bash
+  cordova plugin add https://github.com/aerogear/aerogear-pushplugin-cordova.git
+  cordova plugin add https://github.com/aerogear/aerogear-pushplugin-cordova.git#:/tests
```
Comment (danielpassos) [inline README.md]: s/aerogear-pushplugin-cordova/aerogear-cordova-push

**Comment by danielpassos** [comment]
@edewit Am I doing something wrong?

```shell
[passos]: /tmp/t  
→ cordova create push-test
Creating a new cordova project with name "HelloCordova" and id "io.cordova.hellocordova" at location "/private/tmp/t/push-test"

[passos]: /tmp/t  
→ cd push-test/

[passos]: /tmp/t/push-test  
→ cordova plugin add https://github.com/aerogear/aerogear-cordova-push.git
Fetching plugin "https://github.com/aerogear/aerogear-cordova-push.git" via git clone

[passos]: /tmp/t/push-test  
→ cordova plugin add https://github.com/aerogear/aerogear-cordova-push.git#:/tests
Fetching plugin "https://github.com/aerogear/aerogear-cordova-push.git" via git clone
Error: ENOENT, no such file or directory '/var/folders/8p/ft9nhcyj3k35fr_9kbjmwrvw0000gn/T/plugman/git/1427469858940/tests/plugin.xml'
    at Error (native)
    at Object.fs.openSync (fs.js:502:18)
    at Object.fs.readFileSync (fs.js:354:15)
    at Object.module.exports.parseElementtreeSync (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/util/xml-helpers.js:121:27)
    at /usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/plugman/util/plugins.js:88:35
    at _fulfilled (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:787:54)
    at self.promiseDispatch.done (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:816:30)
    at Promise.promise.promiseDispatch (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:749:13)
    at /usr/local/lib/node_modules/cordova/node_modules/q/q.js:557:44
    at flush (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:108:17)
```

**Comment by edewit** [comment]
seems like a connection failure, could be due to github ddos, could you try it again?

**Comment by danielpassos** [comment]
:+1: 

screenshot 2015-03-30 10 49 59

**Comment by danielpassos** [comment]
Landed at bbdef00c7cc66580506bf90acec1bdcfd297a6b7

**Pull request closed without merging**